tests/functional/TestErrorHandling/tests/brick_evict_test.go

```diff
@@ -272,6 +273,85 @@ func TestBrickEvict(t *testing.T) {
 			"expected device used aggregate sizes to match", newAggUsed, aggUsed)
 		checkConsistent(t, heketiServer)
 	})
+
+	t.Run("lvremoveErrorRecover", func(t *testing.T) {
+		resetConf()
+
+		vinfo1, err := heketi.VolumeCreate(volReq)
+		tests.Assert(t, err == nil, "expected err == nil, got:", err)
+		aggUsed := deviceAggregateUsedSize(t)
+
+		UpdateConfig(origConf, heketiServer.ConfPath, func(c *config.Config) {
+			c.GlusterFS.Executor = "inject/ssh"
+			c.GlusterFS.InjectConfig.CmdInjection.CmdHooks = inj.CmdHooks{
+				inj.CmdHook{
+					Cmd: ".*lvremove.*",
+					Reaction: inj.Reaction{
+						Err: "thwack!",
+					},
+				},
+			}
+		})
+		testutils.ServerRestarted(t, heketiServer)
+
+		toEvict := vinfo1.Bricks[0].Id
+		err = heketi.BrickEvict(toEvict, nil)
+		tests.Assert(t, err != nil, "expected err == nil, got:", err)
```

Comment:
The string parameter needs change, expected err is not nil.

Reply:
Oops. Copy-n-paste-o.